gtkfi/ArcSDM
# Add MLP Regressor Tool to ArcSDM Toolbox

# Description
This pull request introduces the `TrainMLPRegressorTool` to the ArcSDM toolbox. The new tool allows users to train a Multilayer Perceptron (MLP) regressor using their input data, providing a powerful method for predictive modeling and analysis.

# Changes

## 1. New Tool Class:
- Added the `TrainMLPRegressorTool ` class to the ArcSDM toolbox.
- Defined the tool's parameters, including input features, target labels, and various training configurations.

2. Parameter Definitions:
- Defined parameters for input features, target labels, validation method, metrics, split size, cross-validation folds, penalty, maximum iterations, solver, verbosity, random state, and output file.

## Testing 
- Use multiband data to check Multiband reading
- Use WofE FGDB data and input labels for MLP
- Use WofE FGDB data for PCA
- Use other data (FeatureLayer or FeatureClass) as input
